src/utils/testing.ts

```diff
@@ -85,11 +86,13 @@ async function testInContext(code: string, options: TestOptions): Promise<TestRe
     resultStatus: result.status,
     result: result.status === 'finished' ? result.value : undefined
   })
+  const UseLazy = options.lazyEvaluation ? options.lazyEvaluation : false
```

Comment:
Follow the naming convention:
`useLazy`, not `UseLazy`

Here, use `!!` to convert a value to `boolean`:
`const UseLazy = !!options.lazyEvaluation`

https://stackoverflow.com/questions/20093613/typescript-conversion-to-boolean

Reply:
why don't you use `useLazyEval` instead of `useLazy` to make the coding grammar more consistent?

Reply:
That makes sense, I just didn't come up with a good name, let me change it.